clients/runner/src/runner.rs

```diff
@@ -183,6 +183,7 @@ impl Runner {
 			// The set permissions are: -rwx------
 			.mode(0o700)
 			.create(true)
+			.truncate(true)
```

Comment:
I think this is a good suggestion. I just checked at the docs for this option and it will essentially ensure that the opened file is clean, which is what we want since we will save the new binary in there.

In any case at the end of the operation the checksums will be compared, just in case.